SQLTools.sublime-settings

```diff
@@ -2,6 +2,8 @@
 // that you wish to change in your Users/SQLTools.sublime-settings.
 // Don't copy-paste the entire config.
 {
+    "debug" : false,
+
```

Comment:
Why reorder and move this one to the top?

Reply:
Because is it important. By default it come as true, but the first thing I did was set it to false.

Reply:
As you can see, from now on the default is "false", so there is no need for it to be at the top.

Reply:
> of a low interest to an average end user or they are not working properly with all DB's

I do not know much about the average end user. By my experience, they will just try it once, see it is not working and them install some JetBrains IDE for SQL.

The other end user which go beyond this will try a little more, perhaps even open an issue about it. Then would be nice if the first thing they see is the debug option, so perhaps they can spot by themselves the error on their configuration or what is going wrong.

It was already the first option some days ago https://github.com/mtxr/SQLTools/commit/58c679b0678344328b894b9a304d315227618db9, and I think there was a reason about this. This is the first thing you need to enable when you get in trouble, i.e., something is not working. And getting in trouble is something which is certain to happen as [Murphy's law](https://en.wikipedia.org/wiki/Murphy%27s_law) explains. But if you do not believe in this, them it must be moved it down.

Reply:
I know it was the first one a few days ago and I specifically changed it to be in latter parts of config for a good reason.
As of right now, this option does not provide any real debugging - the only text it outputs is listed below (upon plugin loading).
```
SQLTools (v0.9.7): Logging is active
SQLTools (v0.9.7): Connection timeout set to 60 seconds
SQLTools (v0.9.7): Connection history defined with max size 100
SQLTools (v0.9.7): SQLTools Loaded!
```

Reply:
Now I remember why I disabled the debug. It was showing this `SQLTools (v0.9.7): SQLTools Loaded!` when Sublime Text started. Ok, there is not much importance on this for debugging as for now.